debian/changelog

```diff
@@ -1,3 +1,20 @@
+aad-auth (0.5.2) mantic; urgency=medium
+
+  * Update Go version to 1.21
```

Comment:
I would be more precise the the update is in debian/control for go build dependency telling that the symbol file were already requiring them.

Reply:
So we shouldn't touch the required version in `go.mod`?

Reply:
we do, it should have been part of 0.5.1 upload anyway (actually both, debian/control and this) as the symbol file depends on it. Forget about the debian/control explicitness, but tell in the changelog that this is not a new feature-breakage, but basically something we have already done in the last build.